espnet2/mt/espnet_model.py

```diff
@@ -63,6 +64,7 @@ def __init__(
         self.src_eos = src_vocab_size - 1 if src_vocab_size else None
         self.vocab_size = vocab_size
         self.src_vocab_size = src_vocab_size
+        self.src_ignore_id = src_ignore_id
```

Comment:
I'm afraid that this makes the previous models incompatible.

Reply:
I guess it would, if `src_ignore_id` would register as a torch model parameter and it would be missing from `state_dict` in the previous models. However, it appears to be only a class attribute and `torch.load` does not pay attention to it.

I tested the baseline model from this recipe (`espnet/akreal_lh_small_asr2_e_branchformer_wavlm_large_21_km2k_bpe_rm6k_bpe_ts3k_sp`) on a master code with these two extra lines in `espnet2/mt/espnet_model.py`, and it worked:
```
src_ignore_id: int = -1,
...
self.src_ignore_id = src_ignore_id
```

Please let me know if you meant some other type of incompatibility.